### PR TITLE
doxygen

### PR DESCRIPTION
**Pull request by hoffmannjan**
This is just a doxygen stuff, but please look at It as I'll add to this PR more changes soon.

### CODE REVIEW

**Comment by hoffmannjan** [comment]
added more stuff to this PR, still travis fails on esp chips, will dig into that

File: Adafruit_Si4713.cpp [hunk 1]
```diff
@@ -50,60 +78,77 @@ void Adafruit_Si4713::reset() {
   }
 }
 
-
-//////////////////////////////////////////////////////
-
+/*!
+ *    @brief  Set Property
```
Comment (ladyada) [inline Adafruit_Si4713.cpp]: Set chip property over I2C

File: Adafruit_Si4713.cpp [hunk 2]
```diff
@@ -50,60 +78,77 @@ void Adafruit_Si4713::reset() {
...
-    Serial.print(" = "); Serial.println(value);
+  Serial.print("Set Prop ");
+  Serial.print(property);
+  Serial.print(" = ");
+  Serial.println(value);
 #endif
 }
 
+/*!
+ *    @brief  Sends command to Si4713
```
Comment (ladyada) [inline Adafruit_Si4713.cpp]: Command data should already be stored in `_i2ccommand`

File: Adafruit_Si4713.cpp [hunk 3]
```diff
@@ -198,37 +278,48 @@ void Adafruit_Si4713::beginRDS(uint16_t programID) {
   setProperty(SI4713_PROP_TX_COMPONENT_ENABLE, 0x0007);
 }
 
+/*!
+ *    @brief  Set up default PS strings
```
Comment (ladyada) [inline Adafruit_Si4713.cpp]: Set up the RDS station string

File: Adafruit_Si4713.cpp [hunk 4]
```diff
@@ -198,37 +278,48 @@ void Adafruit_Si4713::beginRDS(uint16_t programID) {
...
+ *    @param  *s
+ *            strings to load
```
Comment (ladyada) [inline Adafruit_Si4713.cpp]: String to load

**Comment by hoffmannjan** [comment]
all is fixed, only don't have permissions to push to this repo and add `gh-pages` :)

**Comment by ladyada** [comment]
ok thanks, please check https://github.com/adafruit/Adafruit-Si4713-Library/issues/1, test the hardware, then bump the version!

**Pull request merged**